packages/codemod/src/utils/prop-utils.ts

```diff
@@ -1,31 +1,129 @@
-    if (
-      attr.type === "JSXAttribute" &&
-      attr.name.type === "JSXIdentifier" &&
-      propsNamesMappingOldToNew[attr.name.name]
-    ) {
-      attr.name.name = propsNamesMappingOldToNew[attr.name.name];
-    }
-  });
+  j(elementPath)
+    .find(JSXOpeningElement)
+    .find(JSXIdentifier)
+    .forEach(identifierPath => {
+      if (propsNamesMappingOldToNew[identifierPath.node.name]) {
+        identifierPath.node.name = propsNamesMappingOldToNew[identifierPath.node.name];
+      }
+    });
 }
 
 /**
  * Checks for whether a prop is used in a JSX element
  */
-export function isPropExists(j: JSCodeshift, elementPath: ASTPath<JSXOpeningElement>, propName: string): boolean {
-  const attributes = elementPath.node?.attributes;
-  if (!attributes) return false;
-  return j(elementPath).find(JSXIdentifier, { name: propName }).size() > 0;
+export function isPropExists(j: JSCodeshift, elementPath: ASTPath<JSXElement>, propName: string): boolean {
+  return j(elementPath).find(JSXOpeningElement).find(JSXIdentifier, { name: propName }).size() > 0;
+}
+
+/**
+ * Finds props on a JSX element
+ */
+export function findProps(j: JSCodeshift, elementPath: ASTPath<JSXElement>, ...propNames: string[]) {
+  return j(elementPath)
+    .find(JSXOpeningElement)
+    .find(JSXAttribute)
+    .filter(attr => {
+      const attrName = attr.node.name.type === "JSXIdentifier" ? attr.node.name.name : "";
+      return propNames.includes(attrName);
+    });
+}
+
+/**
+ * Removes a prop from a JSX element
+ */
+export function removeProp(j: JSCodeshift, elementPath: ASTPath<JSXElement>, ...propsNames: string[]): void {
+  j(elementPath)
+    .find(JSXOpeningElement)
+    .find(JSXAttribute)
+    .forEach(attrPath => {
+      if (attrPath.node.name.type === "JSXIdentifier" && propsNames.includes(attrPath.node.name.name)) {
+        j(attrPath).remove();
+      }
+    });
+}
+
+/**
+ * Gets the computed value of a prop
+ */
+function getPropValue(j: JSCodeshift, prop: JSXAttribute): undefined | boolean | Literal["value"] | string {
+  const value = prop.value;
+
+  // boolean flag value (e.g. <Button disabled />)
+  if (value === null) {
+    return true;
+  }
+
+  // string value (e.g. <Button text="Click me" />)
+  if (value?.type === "Literal") {
+    return value.value;
+  }
+
+  // expression value (e.g. <Button text={...} />).
+  if (value?.type === "JSXExpressionContainer") {
+    if (value.expression.type === "Literal") {
+      // e.g. <Button text={true} /> or <Button text={"text"} />
+      return value.expression.value;
+    }
+  }
+
+  // can be very complex, we'll have to compare strings
+  return value ? j(value).toSource() : undefined;
+}
+
+/**
+ * Checks if a Collection of props have the same value
+ */
+export function propsValueMatch(j: JSCodeshift, propsCollection: Collection<JSXAttribute>): boolean {
+  const values = propsCollection.nodes().map(prop => getPropValue(j, prop));
+  return values.every(value => value === values[0]);
```

Comment:
Maybe I'm missing something, but why do we need to use "every"? This function supposed to compare between only 2 values, am I right? Why not a simple value[0] === value[1]

Reply:
just one of the js ways to make sure all items in array are equal, I do not want a compare function, but to know if all are equal

Reply:
But is it possible to have more then 2 items in this specific array?

Reply:
yep, it can include a lot of props in theory (probably 99.99% of cases it won't, unless we'll encounter some very weird use case, but it's good to keep it generic)